### PR TITLE
implement Gemini (ready)

### PR DESCRIPTION
**Pull request by Swingcloud**
seems like novaexchange have fixed the ESP_DOGE issue, and there is no more ESP in market pairs

### CODE REVIEW

File: spec/exchanges/novaexchange/integration/market_spec.rb [hunk 1]
```diff
@@ -29,21 +29,4 @@
     it { expect(@ticker.timestamp).to_not be nil }
     it { expect(@ticker.payload).to_not be nil }
   end
-
```
Comment (dominicwong617) [inline spec/exchanges/novaexchange/integration/market_spec.rb]: @Swingcloud 

1. they should be in 2 different PRs (but it's fine this time)
2. did you remove the hack in the actual code?

Comment (yihangho) [inline spec/exchanges/novaexchange/integration/market_spec.rb]: @dominicwong617 I did a quick check of the current response from Nova, indeed it has been removed, but they added another edge case: https://novaexchange.com/remote/v2/market/info/ESP2_DONATE_/. It seems like it's not an active market, should be safe to ignore.

Comment (Swingcloud) [inline spec/exchanges/novaexchange/integration/market_spec.rb]: yeah I've remove the hack in actual code

Comment (tmlee) [inline spec/exchanges/novaexchange/integration/market_spec.rb]: i noticed that too and issued a commit for that. Just noticed the discussion here later on
